src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -24,10 +24,10 @@ import type { BottomSheetDefaultBackdropProps } from './types';
 
 const BottomSheetBackdropComponent = ({
   animatedIndex,
-  opacity = DEFAULT_OPACITY,
-  appearsOnIndex = DEFAULT_APPEARS_ON_INDEX,
-  disappearsOnIndex = DEFAULT_DISAPPEARS_ON_INDEX,
-  enableTouchThrough = DEFAULT_ENABLE_TOUCH_THROUGH,
+  opacity: userOpacity,
```

Comment:
could you follow the name convection, it should be `_providedOpacity`.

https://github.com/gorhom/react-native-bottom-sheet/blob/955b7748932ba5ea81d2406c0acf7b612fecbf0e/src/components/bottomSheet/BottomSheet.tsx#L97